internal/capability/operator_install_test.go

```diff
@@ -0,0 +1,99 @@
+package capability
+
+import (
+	"context"
+	"os"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/opdev/opcap/internal/operator"
+	"github.com/operator-framework/api/pkg/operators/v1alpha1"
+	operatorv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("Operator install tests", func() {
+	var csv operatorv1alpha1.ClusterServiceVersion
+	var fakeClient operator.Client
+	var operatorGroupData operator.OperatorGroupData
+	var subscription operator.SubscriptionData
+	var customResources []map[string]interface{}
+	var auditFn auditFn
```

Comment:
You probably don't want these two shared (auditFn and auditCleanupFn) between tests. Those can just be created in the It() clause with :=.